README.md

```diff
@@ -165,10 +165,10 @@ target:
 This will be applied in any process that has (`service_name == "frontend"` OR `service_name == "auth"`) AND `datacenter == "AWS-US-EAST-1"`.
 
 ### Regexp Targeting
-To provide a regular expression for targeting, surround it with a leading and trailing slash `/`. For example:
+To provide a regular expression for targeting, use the ruby regex keyword `!ruby/regexp` followed by your regular expression. For example:
 ```
 target:
-  service_name: /frontend-\d/
+  service_name: !ruby/regexp /frontend-\d/
```

Comment:
Here is how `Pysch` parses it this value:
https://github.com/ruby/psych/blob/master/lib/psych/visitors/to_ruby.rb#L96-L110

Reply:
@ColinDKelley @jebentier It looks like we technically are already open to this potential vulnerability! I can switch our how we are parsing yaml to use `Pysch#safe_load_file` and specify allowing only `Regexp` as a permitted class besides the default permitted classes:
https://github.com/ruby/psych/blob/216f94fb9aacb0754f1dd2257b8d6a61b278a8b2/lib/psych.rb#L279-L288

Reply:
See https://github.com/Invoca/process_settings/blob/6ff07b4f82c52f501556b15ccab0b46140794f55/lib/process_settings/targeted_settings.rb#L76

Reply:
Good point--we are already vulnerable! And because the process_settings are set by developers (not parsed from external input) I think we'll be fine?

If it doesn't add much scope, I really like switching over to `Pysch.safe_load_file` with only `Regexp` permitted. But if that takes more than a couple minutes, feel free to create a ticket for Octothorpe instead.

Reply:
Added: https://github.com/Invoca/process_settings/pull/115/commits/c6d10fe6e7ad91bcc878a5489c345d7817cc3d4e